composer.json

```diff
@@ -6,13 +6,13 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php": "~7.1",
+        "php": "^7.2 || ^8.0",
         "phpstan/phpstan": "^0.12",
         "typo3/cms-core": "^8.7 || ^9.5 || ^10.4 || 11.*.*@dev",
         "typo3/cms-extbase": "^8.7 || ^9.5 || ^10.4 || 11.*.*@dev"
     },
     "require-dev": {
-        "consistence/coding-standard": "^3.8",
+        "consistence-community/coding-standard": "^3.8",
```

Comment:
for now we switch to `consistence-community` packages
https://github.com/consistence/coding-standard/pull/65#issuecomment-791898202